docs/architecture/containerized-lifecycle/docker-devops-workflow/create-ci-cd-pipelines-azure-devops-services-aspnetcore-kubernetes.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Steps in the outer-loop DevOps workflow for a Docker application
+title: Creating CI/CD pipelines in Azure DevOps Services for a .NET application on Containers and deploying to a Kubernetes cluster
```

Comment:
```suggestion
title: Create CI/CD pipelines in Azure DevOps Services for a .NET application on Containers and deploying to a Kubernetes cluster
```

Try to avoid the use of gerunds in titles and headings.